CHANGELOG.rst

```diff
@@ -17,36 +17,37 @@ Added
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - When failover mode is stateful, all manual leader promotions will be consistent:
-  every instance before becoming writable performs `wait_lsn` operation to
+  every instance before becoming writable performs ``wait_lsn`` operation to
   sync with previous one. If consistency couldn't be reached due to replication
   failure, a user could either revert it (promote previous leader), or force
   promotion to be inconsistent.
 - Early logger initialization (for Tarantool > 2.5.0-100, which supports it).
-- Add `probe_uri_timeout` argparse option responsible for retrying
+- Add ``probe_uri_timeout`` argparse option responsible for retrying
   "Can't ping myself" error on startup.
-- "Force leader promotion" action for current leader in webui.
 - New test helper: ``cartridge.test-helpers.etcd``.
-- `rpc_call` supports `on_push` and `on_push_ctx` net.box options.
-- Replicasets leaders will have orange flags instead of green ones while
-  they are read-only.
+- Support ``on_push`` and ``on_push_ctx`` options for ``cartridge.rpc_call()``.
 
```

Comment:
Seems I forgot to add note for https://github.com/tarantool/cartridge/pull/989. Could you add it?